content/posts/irods-development-update-february-2023.md

```diff
@@ -0,0 +1,239 @@
+    - did not receive CZI funding
+    - looking at UNC and/or NSF funding opportunities
+    - working meeting
+        - Thur, February 16, 2023, 10am ET
+
+- [Authentication Working Group](https://github.com/irods-contrib/irods_working_group_authentication)
+
+    - building a new API endpoint to provide a flexible authentication mechanism
+    - new authentication plugins will drive the flow
+    - working meeting
+        - Tues, February 28, 2023, 10am ET
+
+- [S3 Working Group](https://github.com/irods-contrib/irods_working_group_s3)
+
+    - develop tools to present iRODS as S3-compatible storage to existing S3 clients
+    - planning and investigation into custom C++ S3 implementation
+    - working meeting
+        - Fri, March 3, 2023, 3pm ET
+
+- [TRiRODS](https://irods.org/trirods)
+
+    - Wed, February 15, 2023, 4pm ET
+    - Kory Draughn, Chief Technologist
+    - GenQuery2: A richer query interface into the iRODS namespace
+
+- [4.2.12](https://github.com/irods/irods/milestone/38)
+
+    - 26 bugs / 36 open, 105 closed
+
+- [4.3.1](https://github.com/irods/irods/milestone/39)
+
+    - 14 bugs / 33 open, 91 closed
+    - goals
+        - iinit to learn about pam_interactive (new auth framework)
+        - support for HA (single irodsDelayServer defined in the catalog)
+
+- [4.3.2](https://github.com/irods/irods/milestone/40)
+
+    - 35 bugs / 178 open, 0 closed
+
+- [4.3 Backlog](https://github.com/irods/irods/milestone/34)
+
+    - 132 bugs / 278 open, 0 closed
+
+- New Development Work
+
+    - no updates
+
+- Active Development Work
+
+    - [iRODS Server](https://github.com/irods/irods)
+        - working with SURF to officially support pam_interactive auth plugin
+            - targeting 4.3.1 or 4.3.2
+        - fixed bug which allowed metadata attribute values to be set to an empty string
+        - itouch now reports a non-zero return code on error
+        - improved pattern matching support in itree
+        - working to improve logical locking to keep things from getting stuck
+        - implemented test fixture which exposes controls for triggering various errors
+            - not merged yet, but considering it
+        - made full filesystem / dstream library useable in the same source file
+        - implemented simple server-side Catch2 inspired unit testing framework
+        - implemented experimental plugins for testing the new GenQuery parser
+
+    - Build and Packaging
+        - adding boost/Qt5-style build system targets
+        - consistent / better use of CMake
+        - targeting gcc/libstdc++ again and defining minimum requirements
+        - entered early stages of moving externals to use standard packaging tools
+            - rpmbuild and dpkg-buildpackage
+
+    - [iRODS Documentation](https://github.com/irods/irods_docs)
+        - finished first draft of the roadmap (ready for review/voting)
+            - https://gist.github.com/korydraughn/0c20d4d8b3fccf17b0661c674bdf9c7b
```

Comment:
Should I remove the link to the gist?

I'm not sure if it should be public yet.

Reply:
I also just noticed that this should probably not be under this section because it's not a part of docs.irods.org.

Of course, that can change. Thoughts?

Reply:
Let's leave out the gist link for now - until we send it to membership and have the vote.   Mostly since this will be a news/blog post and last longer than the gist is relevant...

Reply:
Removed it.